jobs/elasticsearch/monit

```diff
@@ -1,14 +1,15 @@
-<% if p("elasticsearch.monitor.type") == "process") %>
+<% if p("elasticsearch.monitor.type") == "process" %>
 check process elasticsearch
   with pidfile /var/vcap/sys/run/elasticsearch/elasticsearch.pid
   start program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl start'" with timeout 120 seconds
   stop program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl stop'"
   group vcap
-<% else if p("elasticsearch.monitor.type") == "host" %>
+<% elsif p("elasticsearch.monitor.type") == "host" %>
 check host elasticsearch with address 127.0.0.1
   start program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl start'" with timeout 120 seconds
   stop program "/var/vcap/jobs/elasticsearch/bin/monit_debugger elasticsearch_ctl '/var/vcap/jobs/elasticsearch/bin/elasticsearch_ctl stop'"
   if failed url http://127.0.0.1:9200/_cluster/health
+    <% if not p("elasticsearch.node.allow_master") %>and content = '"status":"green"'<% end %>
```

Comment:
I think this should be  `<% if not p("elasticsearch.node.allow_master") and p("elasticsearch.node.allow_data")%>`  there's no sense on blocking other non-data nodes (like parsers) until the cluster is green.  Parsers specifically can still function when the cluster is yellow.